src/chunk_index.c

```diff
@@ -364,6 +365,25 @@ ts_chunk_index_create_from_constraint(int32 hypertable_id, Oid hypertable_constr
 					   get_rel_name(hypertable_indexrelid));
 }
 
+static Oid
+chunk_index_find_matching(Relation chunk_rel, Relation index)
```

Comment:
Only the index relid is used so I suggest passing that in instead of whole rel:

```suggestion
chunk_index_find_matching(Relation chunk_rel, Oid indexrelid)
```